tokens/cnft-candy-machine/anchor/programs/cnft-candy-machine/src/instructions/allow_list.rs

```diff
@@ -0,0 +1,35 @@
+use anchor_lang::prelude::*;
+
+use crate::state::{
+    AllowListStruct, 
+    Config
+};
+
+#[derive(Accounts)]
+pub struct AllowList<'info> {
```

Comment:
This will at some points run out of memory or out of CU. You could add a check or a hint somewhere for the maximum possible list size and check if the program doesnt run out of CU with the max allow list size.